test/test_module_dda_quant.py

```diff
@@ -112,7 +120,25 @@ def test_MaxQuant_file(self):
             Module().benchmarking(
                 user_input["input_csv"], user_input["input_format"], {}, None
             )
-
+            
+            
+            
+            
+class TestPlot(unittest.TestCase): 
+    """Test if the plots return a figure."""
+    
+    with open("../ProteoBench/results.json") as file:
```

Comment:
This code should be part of a method (whether the test below or a new method, in case it would be reused). I am not sure whether we should use the local `results.json` files for this test (I am not sure if this file should be even part of this repo). In case this local file is broken the test would fail. I would suggest using the actual file `https://github.com/Proteobench/Results_Module2_quant_DDA/blob/main/results.json` the raw file is available at https://raw.githubusercontent.com/Proteobench/Results_Module2_quant_DDA/main/results.json